generators/client/templates/angular/src/main/webapp/app/_app.main.ts

```diff
@@ -26,4 +26,6 @@ if (module['hot']) {
     module['hot'].accept();
 }
 
-platformBrowserDynamic().bootstrapModule(<%=angular2AppName%>AppModule);
+platformBrowserDynamic().bootstrapModule(<%=angular2AppName%>AppModule)
+.then((success) => console.log(`Application started`))
```

Comment:
I think its better to have to catch and throw an error when something goes wrong.